README.md

```diff
@@ -14,7 +14,11 @@ Kubernetes interfaces.
 * [Quick start](http://metal3.io/try-it.html)
 * [Demos](https://www.youtube.com/watch?v=VFbIHc3NbJo&list=PL2h5ikWC8viKmhbXHo1epPelGdCkVlF16&ab_channel=Metal3)
 * [Blog posts](https://metal3.io/blog/index.html)
-* [FAQ](https://metal3.io/blog/index.html)
```

Comment:
Is this change intentional?

Reply:
yes, because it is not a FAQ but rather the same as ` [Blog posts](https://metal3.io/blog/index.html)`

Reply:
That is most probably because there is no way to point to FAQ page directly on that page